C++/4. Advanced-Operators.md

```diff
@@ -151,7 +152,7 @@ true
 true
 ```
 
-**Note: These type of operators are mostly not used like this, and are instead used in while loops, for loops, if statements, and in the conditional operator (all of which will be covered later).**
+**Note: These types of operators are generally not used like this, and are instead used in the while loop, the for loop, the if statement, and in the conditional operator (all of which will be covered later).**
```

Comment:
> and are instead used in **a** while loop, **a** for loop, **an** if statement, or in **a** conditional operator (all of which will be covered later).

I feel like this makes more sense. Make the change if you want to.

Reply:
hey make it '...**or** in a conditional operator...'
The '...' means the words before and after. Anyway just change that and I think that's all.